src/Blubrry/REST/Resource/Social.php

```diff
@@ -49,6 +68,85 @@ public function updateListing($program_keyword, $body) {
     public function getSocial($program_keyword, $params) {
         $path = '/2/social/' . $program_keyword . '/get-social-options.json';
 
+        $social_options = array();
```

Comment:
@lucashillebrandt In some places you are using `[]` to declare an empty array, and in other places you are using `array()`. We need to use only one of these. I would recommend to use `[]` for all the arrays.